packages/ui/components/localize/src/utils/resolveLocaleConfig.js

```diff
@@ -0,0 +1,20 @@
+/**
+ *
+ * @param {string} namespace Namespace for the locale
+ * @param {any} localeDir Directory of the locale
+ * @returns Promise<LocaleConfig>
+ */
+export const resolveLocaleConfig = (namespace, localeDir) => ({
+  [namespace]: (locale = 'en') => {
+    const localePath = (/** @type {string} */ loc) => [localeDir, `${loc}.js`].join('/');
+
+    const fallbackLocale = localePath(locale.replace(/-\w+/, ''));
+
+    return import(localePath(locale)).catch(reason => {
+      console.warn(`
+        Locale ${locale} not found. Reason ${reason}.
+        Loading fallback locale ${fallbackLocale}`);
+      return import(fallbackLocale);
```

Comment:
Can we add 'en' as another if the fallbackLocale does not exists at all?

Reply:
Good idea to have fallback to en. I will add that.